datahub-web-react/src/app/entity/shared/components/styled/DeprecationPill.tsx

```diff
@@ -56,11 +57,14 @@ const StyledInfoCircleOutlined = styled(InfoCircleOutlined)`
 `;
 
 type Props = {
+    urns: Array<string>;
```

Comment:
while the mutation below is batch, I think this pill is only relevant to one entity at a time, so I think this can be a singular `urn`.

Reply:
Checked and Fixed!!